src/cli/cluster/cluster_manager.js

```diff
@@ -85,14 +85,16 @@ module.exports = class ClusterManager {
     const chokidar = require('chokidar');
     const fromRoot = require('../../utils/fromRoot');
 
-    this.watcher = chokidar.watch([
+    const watchPaths = uniq([
       'src/plugins',
```

Comment:
implemented a fix in https://github.com/elastic/kibana/pull/6446, hope to get that merged soon